src/main/java/io/permit/sdk/enforcement/Enforcer.java

```diff
@@ -448,6 +449,63 @@ public UserPermissions getUserPermissions(GetUserPermissionsQuery input) throws
         return result;
     }
 
+
+    @Override
+    public UserPermissions getUserPermissionsWithOPA(GetUserPermissionsQuery input) throws IOException, PermitApiError {
```

Comment:
this doesn't support the FactDB ( because the factdb should be called instead of OPA and the url would be different ), just worth documenting it for now in the docs

Reply:
i will open a pr in the docs